CONTRIBUTING.md

```diff
@@ -1,49 +1,151 @@
++ Le _store_ (magasin, réserve) d'EmberData fournit une interface pour retrouver les _records_ (enregistrements) d'un seul type. Utilisez `store.findRecord()` pour récupérer un _record_.
+```
+
+If the literal translation of a term is _reeeeally_ convoluted, don't hesitate to be wordy about it. The purpose is to make this understandable to non-English speakers:
+
+```diff
+- This method is also known as the _model hook_.
++ Cette méthode est aussi appelée le _model hook_ (_hook_ signifie littéralement "un crochet" mais se traduit mal dans un contexte technique. On peut voir le _hook_ comme une fonction mise à disposition et exécutée en interne par le framework, et dont on définit le contenu, on y "accroche" du code).
+```
+
+### 3. Don't translate articles titles / translate descriptions
+
+Some pages of the Guides reference articles, introduced with a title and sometimes a description. If this resource is also available in French (for instance, a lot of pages from MDN website are translated), then use the French URL in the translation (be careful with that MDN example though, because some pages are not translated entirely and it can be better to keep the English).
+
+If the resource is not translated or if the translation is incomplete, keep the English link and translate only the description, not the title of the resource itself, because we need to make clear to the reader that it's an English document:
+```diff
+- [Using ARIA:](https://www.w3.org/TR/using-aria/) a practical guide for developers on how to add accessibility information to HTML element
++ <!-- spell ignore -->
++ [Using ARIA](https://www.w3.org/TR/using-aria/) : un guide pratique pour ajouter aux éléments HTML des informations accessibles.
+```
+
+### 4. Set non-breakable spaces
+
+`&nbsp;` is the HTML sign for "non-breakable space" and can be used in markdown as well. If the space is not set non-breakable, then a punctuation sign like ":" or ";" could be separated from the previous word and moved to the next line, which is incorrect.
+```diff
+- Using ARIA: a practical guide (...)
++ Using ARIA&nbsp;: un guide pratique (...)
+```
+
+## What tone should I use?
+
+### 1. General tone
+
+To stick to the official Guides tone, try to find the right balance to be "relax" without being too familiar:
+- Prefer "ça" over "cela"
+- Use "nous" when you mean "you & the reader", let's do this together (this is used a lot in the tutorial): "À présent, faisons ça"
+- Use "on" as the undefined pronoun, when you talk about a general rule: "En général, on écrit les variable en anglais"
+
+### 2. Inclusive writing
+
+We generally avoid the "·" character in the translation because all screen readers don't handle it very well. However, there are other ways to be inclusive in your writing. Be aware of all masculine endings and don't hesitate to rephrase the sentence in order to use a more neutral tone.
```

Comment:
```suggestion
We avoid the "·" character in the translation because all screen readers don't handle it very well. However, there are other ways to be inclusive in your writing. Be aware of all masculine endings and don't hesitate to rephrase the sentence in order to use a more neutral tone.
```